storage/innobase/row/row0upd.cc

```diff
@@ -2782,6 +2801,7 @@ row_upd_clust_rec_by_insert(
 			ut_ad(!trx_undo_roll_ptr_is_insert(
 			              row_get_rec_roll_ptr(rec, index,
 							   offsets)));
+			row_upd_reown_inherited_fields(entry);
```

Comment:
The comment above says: "Skip delete marking clustered index and disowning its blobs." I would change it to "re-owning its blobs".

Reply:
We will skip changing `rec`, but we must mark BLOBs in the `entry` as "owned", like they were on the previous call of `row_upd_clust_rec_by_insert()` that had returned `DB_LOCK_WAIT`. I will clarify the comment.